language/stdlib/modules/doc/overview.md

```diff
@@ -6,15 +6,13 @@
 
 This is the root document for the Libra framework module documentation. The Libra framework provides a set of Move
 modules which define the resources and functions available for the Libra blockchain. Each module is individually
-documented here, together with it's implementation and [formal specification](../../../move-prover/doc/user/spec-lang.md).
+documented here, together with it's implementation and
```

Comment:
That should be "its", not "it's" (no apostrophe)

Reply:
Done